Source/Shaders/ShadowVolumeVS.glsl

```diff
@@ -0,0 +1,82 @@
+attribute vec3 positionHigh;
+attribute vec3 positionLow;
+attribute vec3 normal;
+attribute float extrude;
```

Comment:
Perhaps we can pack this into the direction of the normal, e.g., normals facing "down" indicate vertices that are dynamically extruded downward.